### DISCUSSION TITLE
View species level checklist frequency?

### DISCUSSION THREAD

**Discussion by timbean** [mcguirepr89/BirdNET-Pi#1163] [Q&A] [answered]
Hi, is there an easy way to view a species specific checklist frequency that's used in the BirdNET model to remove unlikely occurrences? I'm trying to figure out why my BirdNET PI is hearing and classifying cedar waxwings at a probability greater than the minimum but not saving the files or recording the occurrence. Thanks!

**Comment by rabede** [comment, marked as answer]
Hi, if you click on "more info" and then "Species List Tester" on the Basic Settings page under Model you get a list with the occurence Frequency.

**Reply by timbean** [reply]
Perfect, thanks so much. Are there other discussions I missed on this? I'd be curious to hear other peoples' experiences balancing this vs. the sensitivity/specificity settings.
